src/main/java/net/sf/mzmine/modules/peaklistmethods/io/siriusexport/SiriusExportParameters.java

```diff
@@ -46,9 +52,21 @@
                     new String[] {ROUND_MODE_MAX, ROUND_MODE_SUM},
                     ROUND_MODE_MAX);
     
-    public static final MassListParameter MASS_LIST = new MassListParameter();
+    public static final MassListParameter MASS_LIST = new MassListParameter();          
+    
+    public SiriusExportParameters() {    	
+    	super(new Parameter[] {PEAK_LISTS, FILENAME, ROUND_MODE, MASS_LIST});
+    }
     
-    public SiriusExportParameters() {
-	super(new Parameter[] {PEAK_LISTS, FILENAME, /*FRACTIONAL_MZ,*/ ROUND_MODE, MASS_LIST});
+    public ExitCode showSetupDialog(Window parent, boolean valueCheckRequired) { 
+    	String message = "<html>SIRIUS Module Disclaimer:" +
+        		"<br>    - If you use the SIRIUS export module, cite MZmine2 paper and the following articles: Dührkop et al.," +
```

Comment:
Actually, both citations are needed. Indeed Sirius 3.5 contains, Sirius algorithm (molecular formula from fragmentation tree), and CSIFingerID tool for structure search.